### PR TITLE
Add s3 migration base system test

### PR DESCRIPTION
**Pull request by stewartie4**
actions: https://github.com/0chain/actions/pull/137

### CODE REVIEW

File: tests/cli_tests/s3mgrt_migrate_test.go [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+)
+
+func TestS3Migration(testSetup *testing.T) {
+	t := test.NewSystemTest(testSetup)
+
+	if s3SecretKey == "" || s3AccessKey == "" {
+		t.Skip("s3SecretKey or s3AccessKey was missing")
+	}
+
+	t.RunSequentiallyWithTimeout("Should migrate existing bucket successfully", 15*time.Minute, func(t *test.SystemTest) {
```
Comment (Kishan-Dhakan) [inline tests/cli_tests/s3mgrt_migrate_test.go:20]: 15 minutes is a long time👀. What options do we want to run this? Starting a thread on slack

**Review by Kishan-Dhakan — CHANGES_REQUESTED** [review]
That's a great cornerstone PR for these stories! Happy to merge post green run but would like to discuss how we go about running these cases on slack.

File: tests/cli_tests/s3mgrt_migrate_test.go (hunk 1 above)
Comment (stewartie4) [reply tests/cli_tests/s3mgrt_migrate_test.go]: created a smaller bucket and now the test runs in ~20s

**Comment by stewartie4** [comment]
system tests passing, minus known authorizor crash

**Pull request merged**